examples/transactions-distributed/tests/transactions_distributed_test.bal

```diff
@@ -46,7 +45,6 @@ function testFunc() {
     test:assertTrue(foundMatch(untaint outputs, "Got response from bizservice"));
 
     test:assertTrue(foundMatch(untaint outputs,  "Initiated transaction committed"));
-    test:assertTrue(foundMatch(untaint outputs, "Sent response back to client"));
```

Comment:
Since we can't guarantee the happens before relationship between initiator (initiator logging  "Sent response back to client" message) and test case executing this assertion. To avoid intermittent test failure due to "happens before" violation, only option I see is to remove the check on this line.

I think this is fine because this logged message "Sent response back to client" has nothing to do with "distributed-transactions" logic.